bundles/org.openhab.binding.miio/README.md

```diff
@@ -2806,6 +2847,58 @@ Number SpeedLevel "Fan-Speed Level" (G_fan) {channel="miio:basic:fan:SpeedLevel"
+
+```java
+Group G_gateway "Mi Smart Home Gateway v1" <status>
+Switch telnetEnable "Enable Telnet" (G_gateway) {channel="miio:basic:gateway:telnetEnable"}
+Number doorbelVol "Doorbell Volume" (G_gateway) {channel="miio:basic:gateway:doorbelVol"}
+Number gatewayVol "Gateway Volume" (G_gateway) {channel="miio:basic:gateway:gatewayVol"}
+Number alarmingVol "Alarming Volume" (G_gateway) {channel="miio:basic:gateway:alarmingVol"}
+String doorbelPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbelPush"}
+```
+
+### Mi Smart Home Gateway v2 (lumi.gateway.v2) item file lines
+
+note: Autogenerated example. Replace the id (gateway) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
+
+```java
+Group G_gateway "Mi Smart Home Gateway v2" <status>
+Switch telnetEnable "Enable Telnet" (G_gateway) {channel="miio:basic:gateway:telnetEnable"}
+Number doorbelVol "Doorbell Volume" (G_gateway) {channel="miio:basic:gateway:doorbelVol"}
+Number gatewayVol "Gateway Volume" (G_gateway) {channel="miio:basic:gateway:gatewayVol"}
+Number alarmingVol "Alarming Volume" (G_gateway) {channel="miio:basic:gateway:alarmingVol"}
+String doorbelPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbelPush"}
+```
+
+### Mi Smart Home Gateway v3 (lumi.gateway.v3) item file lines
+
+note: Autogenerated example. Replace the id (gateway) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
+
+```java
+Group G_gateway "Mi Smart Home Gateway v3" <status>
+Switch telnetEnable "Enable Telnet" (G_gateway) {channel="miio:basic:gateway:telnetEnable"}
+Number doorbelVol "Doorbell Volume" (G_gateway) {channel="miio:basic:gateway:doorbelVol"}
+Number gatewayVol "Gateway Volume" (G_gateway) {channel="miio:basic:gateway:gatewayVol"}
+Number alarmingVol "Alarming Volume" (G_gateway) {channel="miio:basic:gateway:alarmingVol"}
+String doorbelPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbelPush"}
+```
+
+### Xiaomi Mi Mijia Gateway V3 ZNDMWG03LM (lumi.gateway.mgl03) item file lines
+
+note: Autogenerated example. Replace the id (gateway) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
+
+```java
+Group G_gateway "Xiaomi Mi Mijia Gateway V3 ZNDMWG03LM" <status>
+Switch telnetEnable "Enable Telnet" (G_gateway) {channel="miio:basic:gateway:telnetEnable"}
+Number doorbelVol "Doorbell Volume" (G_gateway) {channel="miio:basic:gateway:doorbelVol"}
+Number gatewayVol "Gateway Volume" (G_gateway) {channel="miio:basic:gateway:gatewayVol"}
+Number alarmingVol "Alarming Volume" (G_gateway) {channel="miio:basic:gateway:alarmingVol"}
+String doorbelPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbelPush"}
```

Comment:
You should also fix this spelling mistake everywhere else as well.
```suggestion
String doorbellPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbellPush"}
```

Reply:
thanks. fixed.